persistent-qq/persistent-qq.cabal

```diff
@@ -55,7 +55,7 @@ test-suite specs
     , hspec
     , monad-logger
     , mtl
-    , persistent >=2.10
+    , persistent >=2.13
```

Comment:
hackage revision was made.

2.12.0.1 wasn't released yet so I'm going to be releasing it under that.